Testing/functional/tests/cypress/integration/e2e/timeline/mobile/clinicalDocument.js

```diff
@@ -15,17 +15,20 @@ describe("Clinical Document", () => {
     it("Validate Card Details for Mobile", () => {
         cy.get("[data-testid=timelineCard]").last().scrollIntoView().click();
 
-        const entryDetails = cy.get("[data-testid=entryDetailsCard]");
-        cy.get("[data-testid=backBtn]").should("be.visible");
-        entryDetails
-            .get("[data-testid=entryCardDetailsTitle]")
-            .should("be.visible");
-        entryDetails.get("[data-testid=entryCardDate]").should("be.visible");
-        entryDetails
-            .get("[data-testid=clinical-document-discipline")
-            .should("be.visible");
-        entryDetails
-            .get("[data-testid=clinical-document-facility")
-            .should("be.visible");
+        cy.get("[data-testid=entryDetailsCard]")
```

Comment:
This seems very generic.  Is there only one record or are there more?  Is this coming from seed data?  Or has this been created?